src/components/address-bar/styles.scss

```diff
@@ -1,15 +1,16 @@
 @use '../../shared-components/theme-engine/theme' as theme;
 
 .address-bar {
-  background-color: theme.$background-color-primary;
+  background-color: theme.$address-bar-background-color-outer;
```

Comment:
i know it's not always possible, but let's try to avoid having theme values tied to the app structure like this. if we need to name the colors in a more descriptive way than primary/secondary, we can do so, but the more the theme is tied to the actual implementation of the app, the harder it is to change things. both for us, and the designers.

if the designs are not organizing the colors well, we can definitely push back on them, and ask them to be a bit more consistent. it is better for everybody if the palette we are using is as small as possible, and is as consistent across the app as possible. eg. if they're using more than 3 font colors (aside from logos/branding, etc.), we should check in with them, as that will start to get pretty unwieldy. if they really want to use more, then we should at least work with them to define what the role of those additional colors will be.